packages/ui-new/src/Text/Text.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { cva, type VariantProps } from "class-variance-authority";
+import { cn } from "~/utils";
+
+export type TextTags = "span" | "div";
+
+const textVariants = cva("font-sans font-normal", {
+    variants: {
+        size: {
+            xl: "text-xl leading-normal",
+            lg: "text-base leading-normal",
+            md: "text-sm leading-normal",
```

Comment:
I guess we'll also need those variants, like `muted`? I guess you wanted to leave this after the call with Kreso?

Reply:
Exactly, I'm leaving all the color-related variants behind